packages/x-charts/src/ChartsTooltip/utils.tsx

```diff
@@ -15,83 +15,70 @@ export type UseMouseTrackerReturnValue = null | MousePosition;
  * @deprecated We recommend using vanilla JS to let popper track mouse position.
  */
 export function useMouseTracker(): UseMouseTrackerReturnValue {
-  const svgRef = useSvgRef();
+  const { instance } = useChartContext();
 
   // Use a ref to avoid rerendering on every mousemove event.
   const [mousePosition, setMousePosition] = React.useState<MousePosition | null>(null);
 
   React.useEffect(() => {
-    const element = svgRef.current;
-    if (element === null) {
-      return () => {};
-    }
-
-    const controller = new AbortController();
-
-    const handleOut = (event: PointerEvent) => {
-      if (event.pointerType !== 'mouse') {
-        setMousePosition(null);
-      }
-    };
-
-    const handleMove = (event: PointerEvent) => {
-      setMousePosition({
-        x: event.clientX,
-        y: event.clientY,
-        height: event.height,
-        pointerType: event.pointerType as MousePosition['pointerType'],
-      });
-    };
-
-    element.addEventListener('pointerdown', handleMove, { signal: controller.signal });
-    element.addEventListener('pointermove', handleMove, { signal: controller.signal });
-    element.addEventListener('pointerup', handleOut, { signal: controller.signal });
+    // Drag event is triggered by mobile touch or mouse drag.
+    const positionOnDragHandler = instance.addMultipleInteractionListeners(
+      ['move', 'drag'],
```

Comment:
It looks like a hack to have the `use-gesture` whereas the `pointermove` event is exactly what we are looking for

Reply:
Seems like useGesture also support native events
https://use-gesture.netlify.app/docs/gestures/#native-event-handlers-in-react

Reply:
In this case it is probably better to use `pointerMove` indeed